src/pybamm/models/full_battery_models/lithium_ion/basic_ecm_split_OCV.py

```diff
@@ -0,0 +1,100 @@
+#
+# Equivalent Circuit Model with split OCV
+#
+import pybamm
+
+
+class ECMsplitOCV(pybamm.BaseModel):
+    """Basic Equivalent Circuit Model that uses two OCV functions
+    for each electrode. This model is easily parameterizable with minimal parameters.
+    This class differs from the :class: pybamm.equivalent_circuit.Thevenin() due
+    to dual OCV functions to make up the voltage from each electrode.
+
+    Parameters
+    ----------
+    name: str, optional
+        The name of the model.
+    """
+
+    def __init__(self, name="ECM with split OCV"):
+        super().__init__(name)
+
+        ######################
+        # Variables
+        ######################
+        # All variables are only time-dependent
+        # No domain definition needed
+
+        c_n = pybamm.Variable("Negative particle SOC")
+        c_p = pybamm.Variable("Positive particle SOC")
```

Comment:
The standard we use is `theta` for the dimensionless stoichiometry (that goes from 0 to 1) as opposed to `c` for the dimensional concentration (that goes from 0 to c_max), and at the electrode level we call it "stoichiometry" instead of "SOC" to avoid confusion with full cell SOC

Reply:
Got it. I updated this within the code.